crates/task-impls/src/quorum_proposal_recv/handlers.rs

```diff
@@ -72,62 +69,38 @@ async fn validate_proposal_liveness<TYPES: NodeType, I: NodeImplementation<TYPES
         warn!("Couldn't store undecided state.  Error: {:?}", e);
     }
 
-    let liveness_check = justify_qc.view_number() > consensus_write.locked_view();
```

Comment:
How come we're removing this logic? I don't see it replaced anywhere.

Reply:
I wanted to move the `QuorumProposalLivenessValidated` emission, but should have kept this liveness check. Restored!